packages/redux/src/contents/reducer.ts

```diff
@@ -14,6 +14,11 @@ export const INITIAL_STATE_CONTENT: ContentsState = {
     isLoading: {},
     result: null,
   },
+  seoFiles: {
+    error: {},
+    isLoading: {},
+    result: null,
```

Comment:
```suggestion
    result: undefined,
```

Reply:
The values above, like `metadata` and `contentTypes` have the result as `undefined`. It isn´t better to leave it as `undefined` to uniformization the data?

Reply:
Both of them should be `undefined` as well. It was not detected before because the hook that uses them was performing additional code to make sure that when `null` is received, `data` would be `undefined`. You can change those, you might need to fix some types as well but it is okay to change it to `undefined`.

Reply:
Resolved! ✅